app/src/main/res/values/strings.xml

```diff
@@ -20,6 +20,9 @@
     <!-- No Open Tabs Message Description -->
     <string name="no_open_tabs_description">Your open tabs will be shown here.</string>
 
+    <!-- About content. The first parameter is the name of the application. (For example: Fenix) -->
+    <string name="about_content">%1$s is produced by Mozilla.</string>
```

Comment:
Should be Mozilla hard-coded? What about anyone else redistributing the code under a different brand?